radicle-httpd/src/api/test.rs

```diff
@@ -0,0 +1,135 @@
+use std::path::Path;
+use std::sync::Arc;
+use std::{env, fs};
+
+use axum::body::Body;
+use axum::http::Request;
+use axum::response::Response;
+use axum::Router;
+use rand::distributions::{Alphanumeric, DistString};
+use serde_json::Value;
+use tower::ServiceExt;
+
+use radicle::cob::issue::Issues;
+use radicle::git::raw as git2;
+use radicle::storage::WriteStorage;
+use radicle_cli::commands::rad_init;
+use radicle_crypto::ssh::keystore::MemorySigner;
+use radicle_crypto::Signer;
+
+use crate::api::Context;
+
+pub const HEAD: &str = "1e978d19f251cd9821d9d9a76d1bd436bf0690d5";
+pub const HEAD_1: &str = "f604ce9fd5b7cc77b7609beda45ea8760bee78f7";
```

Comment:
Where do these come from?

Reply:
they are hash of commits generated from our tree, it's stable and I've extracted them here to make further changes easier, e.g. if we change "fixture", like name of repo or sig or anything, we only update HEADs here and then good to go.